test/test_ops.py

```diff
@@ -306,7 +306,10 @@ def _test_consistency_helper(samples, variants):
                     cloned = clone_input_helper(sample.input) if variant in inplace_ops else sample.input
 
                     if variant in inplace_ops and sample.broadcasts_input:
-                        with self.assertRaises(RuntimeError):
+                        with self.assertRaises(RuntimeError,
+                                               msg=('inplace variant either allowed '
```

Comment:
Language looks good overall, suggested tweak:

"inplace variant either incorrectly allowed..."

Just to confirm this is a bug if it occurs